packages/next-ui/Overlay/components/OverlayBase.tsx

```diff
@@ -469,7 +469,7 @@ export function OverlayBase(incomingProps: LayoutOverlayBaseProps) {
             })}
           >
             <LayoutProvider scroll={scrollYOffset}>
-              {active && (typeof children === 'function' ? children() : children)}
+              {typeof children === 'function' ? children() : children}
```

Comment:
Something like this?
 
```suggestion
              {typeof children === 'function' ? (active && children()) : children}
```

Reply:
Fixed